### ISSUE TITLE
Add ability to train on negative segmentation examples

### ISSUE THREAD

**Issue by arcrank** [pytorch/vision#1425]
Most use cases for training segmentation networks require both an instance of a _positive_ and _negative_ class. However, certain problems such as a binary detection and segmentation of multiple instances of the same class would benefit from the ability to feed network negative examples so it knows what not to select.  For the mask rcnn torchvision implementation I cannot find way to do this.  Similar repos suggest initializing empty boxes and target variables in dataset for dataloader however I'm getting errors with torch trying to do operations on empty vectors.  Ideally, the loss would be 0 for not having any detections and otherwise nonzero.  I believe that this is something that would be beneficial to be integrated or would love to know if anyone has any work arounds.  I am not looking to exclude these images, I would like to specifically reduce false detections on them.

**Comment by fmassa** [comment]
We've added support for this in the 0.6.0 release of torchvision, see https://github.com/pytorch/vision/releases/tag/v0.6.0
